lib/std/math.zig

```diff
@@ -1431,11 +1431,6 @@ test "minInt and maxInt" {
     try testing.expect(minInt(i128) == -170141183460469231731687303715884105728);
 }
 
-test "max value type" {
-    const x: u32 = maxInt(i32);
```

Comment:
Note the different types `u32` on the variable and `i32` as a parameter.
This test checks if the return type of `maxInt(i32)` can be coerced to a different type `u32`, essentially asserting that it is a `comptime_int` or `u31`.
So I would say that this test is different from the referenced `expect(maxInt(i32) == 2147483647)`